### ISSUE TITLE
Fixing implicit API reference in Metadata Export documentation

### ISSUE THREAD

**Issue by pkiraly** [IQSS/dataverse#6178]
The Admin Guide's Metadata Export page (http://guides.dataverse.org/en/4.16/admin/metadataexport.html) contains the following statement:

```
The following 2 API calls are provided:

/api/admin/metadata/exportAll

/api/admin/metadata/reExportAll
```

According to the "Don't Make Me Think" principle I prefer to give a full URL in the documentation, so http://localhost:8080/api/admin/metadata/exportAll, and http://localhost:8080/api/admin/metadata/reExportAll or even full commands, like

```
curl http://localhost:8080/api/admin/metadata/exportAll

curl http://localhost:8080/api/admin/metadata/reExportAll
```

**Comment by pdurbin** [comment]
@pkiraly yes! @donsizemore and I have talked a lot about how full examples with curl are better than just "PUT" or whatever and that's what #6083 is about. Would you like to help with that effort? :smile:

**Comment by pkiraly** [comment]
Sure

**Comment by pdurbin** [comment]
@pkiraly thanks! Please note that I'm using a new syntax with "export" commands. Please see http://guides.dataverse.org/en/4.16/api/getting-started.html#curl-examples-and-environment-variables and let me know if you have any questions!

**Comment by pdurbin** [comment]
Fixed in pull request #6179. Closing. Thanks, @pkiraly !
